.github/workflows/update-project.yml

```diff
@@ -3,9 +3,11 @@ on:
   push:
     branches:
       - master # Every time a PR is merged to master
+      - feature/reorg # Also feature/reorg, temporarily
```

Comment:
Can you confirm that packages right now should fail since we have a `composer.json` file but have not specifically set the mirror-repo value? I'd like to make sure we're only updating the `jetpack-production` repo out of `feature/reorg` and not any packages.

Reply:
It falls back to the `name` field in composer.json if `extras.mirror-repo` isn't set.

But when it pushes from feature/reorg, it'll push to a feature/reorg branch in the mirror repos instead of to the mirror repos' master.

Reply:
Ah, I see the .name bit now. 👍 

I like the idea of declaring it, though, to prevent accidental overwriting of an unrelated repo if it exists.